modules/javafx.graphics/src/main/java/com/sun/javafx/application/PlatformImpl.java

```diff
@@ -1060,5 +1060,4 @@ private static void checkHighContrastThemeChanged(Map<String, Object> preference
             setAccessibilityTheme(null);
         }
     }
-
```

Comment:
Please revert all changes to this file, since you now aren't touching it in your PR.